src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/query-time-range.mdx

```diff
@@ -0,0 +1,2701 @@
+
+  Example query: `FROM Foo SELECT * SINCE 1702899296789 UNTIL 1702902896789`
+
+  These timestamp values resolve to a time range starting at
+  <span style="font-family:monospace">2023-12-18T11:34:56.789Z</span>, and ending
+  at  <span style="font-family:monospace">2023-12-18T12:34:56.789Z</span>.
+
+### Relative time: <InlineCode>*n* \[time units] ago</InlineCode> [#relative-time-n-units-ago]
+Relative time can be referenced as <InlineCode>*n* [time units] ago</InlineCode>.
+The time is relative to the time the query is executed.
+
+  * <InlineCode>*n*</InlineCode>: A positive integer value
+  * `[time units]`: Available time units include `seconds`, `minutes`, `hours`, `days`, `weeks`,
+`months` (A month is defined as 30 days), `quarters` (A quarter is defined as 91 days),
+`years` (A year is defined as 365 days).
+Time unit pluralization is optional, so you can leave the "s" off the end of the unit.
+`2 day ago` and `2 days ago` have the same meaning.
+
+<Collapser title="Relative time examples">
+  In these examples, assume the query was executed at <span style="font-family:monospace">2023-12-18T12:34:56.789Z</span>.
+  <table style={{ width: "auto" }}>
+    <thead>
+    <tr>
+      <th>
+        Relative time reference
+      </th>
+      <th>
+        Resolved time (UTC)
+      </th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+      <td>`NOW` (query execution time)</td>
+      <td style="font-family:monospace">2023-12-18T12:34:56.789Z</td>
+    </tr>
+    <tr>
+      <td>`2 seconds ago`</td>
+      <td style="font-family:monospace">2023-12-18T12:34:**54**.789Z</td>
+    </tr>
+    <tr>
+      <td>`2 minutes ago`</td>
+      <td style="font-family:monospace">2023-12-18T12:**32**:56.789Z</td>
+    </tr>
+    <tr>
+      <td>`2 hours ago`</td>
+      <td style="font-family:monospace">2023-12-18T**10**:34:56.789Z</td>
+    </tr>
+    <tr>
+      <td>`2 days ago`</td>
+      <td style="font-family:monospace">2023-12-**16**T12:34:56.789Z</td>
+    </tr>
+    <tr>
+      <td>`2 weeks ago`</td>
+      <td style="font-family:monospace">2023-12-**04**T12:34:56.789Z</td>
+    </tr>
+    <tr>
+      <td>`2 months ago`</td>
+      <td style="font-family:monospace">2023-**10-19**T12:34:56.789Z</td>
+    </tr>
+    <tr>
+      <td>`2 quarters ago`</td>
+      <td style="font-family:monospace">2023-**06-19**T12:34:56.789Z</td>
+    </tr>
+    <tr>
+      <td>`2 years ago`</td>
+      <td style="font-family:monospace">**2021**-12-18T12:34:56.789Z</td>
+    </tr>
+    </tbody>
+  </table>
+</Collapser>
+
+### Relative temporal time [#relative-temporal-time]
+Relative time can also be specified using temporal time references.
+These relative time references resolve to the beginning of the temporal time interval.
+Relative temporal time reference will honor a `WITH TIMEZONE` clause.
+
+<Collapser title={<>Day references - <InlineCode>today</InlineCode>, <InlineCode>yesterday</InlineCode> or a weekday</>}>
```

Comment:
Why the <> and </>?

Reply:
It won't work without them. I have to use {<>....</>} here instead of "..." so I can include <InlineCode> tags in the content.